src/pages/costModels/createCostModelWizard/steps.tsx

```diff
@@ -146,6 +168,12 @@ export const validatorsHash = {
     () => true,
     () => true,
   ],
+  IBM: [
+    ctx => nameErrors(ctx.name) === null && descriptionErrors(ctx.description) === null && ctx.type !== '',
+    ctx => ctx.markup !== '' && !isNaN(Number(ctx.markup)),
+    () => true,
+    () => true,
+  ],
```

Comment:
I don't think we should update cost models now.

We don't have anyway to hide this from production.

Reply:
done